### PR TITLE
Multiple django versions

### PR DESCRIPTION
**Pull request by morenol**
Use tox to control the Django version used in the tests.

### CODE REVIEW

**Comment by morenol** [comment]
I think that we can use something like this in order to test this with multiple versions of django. 

What do you think? @mariajgrimaldi @felipemontoya 

I think that the django22 are failing due to a problem with django-fake-model

It seems that django-fake-model is not longer maintained and in the last version, it does not have tests for django22:

https://github.com/erm0l0v/django-fake-model/blob/master/.travis.yml

**Comment by mariajgrimaldi** [comment]
This can be really useful to check errors like django-fake-model with django22. I've been having problems with django-fake-model and this tells me that it is better to stop using it to prevent problems in the future.  Before merging this PR I should solve that issue so the tests pass @morenol

**Comment by morenol** [comment]
> Before merging this PR I should solve that issue so the tests pass @morenol 

Sure

**Comment by morenol** [comment]
@mariajgrimaldi I removed the django22 test, I think that we can merge this and once we are ready we can add the django22 test in another PR.

**Comment by mariajgrimaldi** [comment]
> @mariajgrimaldi I removed the django22 test, I think that we can merge this and once we are ready we can add the django22 test in another PR.

Nice!! That's better. Thank you

**Comment by mariajgrimaldi** [comment]
Can we merge this? @felipemontoya

**Comment by felipemontoya** [comment]
I see nothing here that makes me think we should not merge. However I also have not performed a real review.
@morenol @mariajgrimaldi @andrey-canon the question is up to you?

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,31 +46,35 @@ jobs:
...
       - run:
           name: Run tests
           command: |
             . venv/bin/activate
-            coverage run --source="." manage.py test
-            coverage report --fail-under=71 -m
-            pycodestyle ./eox_tagging
-            pylint ./eox_tagging --rcfile=./setup.cfg
-            isort --check-only --recursive --diff ./eox_tagging
+            tox -e ${TOX_ENV//./} -- coverage run --source="." manage.py test
```
Comment (andrey-canon) [inline .circleci/config.yml:64]: why not make run-test ??? @morenol

Comment (morenol) [reply .circleci/config.yml]: It was not being used before. But now I added that too.

**Pull request merged**